test/models/shipit/review_stack_test.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module Shipit
+  class ReviewStackTest < ActiveSupport::TestCase
+    test ".review_request is nil by default" do
+      stack = shipit_stacks(:review_stack)
+      assert_nil stack.review_request
+    end
+
+    test ".review_request returns nil when all pull requests are merge requests" do
+      stack = shipit_stacks(:review_stack)
+
+      assert_nil stack.review_request
+    end
+
+    test ".review_request returns latest non merge request" do
+      stack = shipit_stacks(:review_stack)
+      pull_request = PullRequest.create!(stack: stack, number: "1", review_request: true)
+
+      assert stack.review_request, pull_request
+    end
+
+    test "automatically enqueues a for provisioning only on create" do
```

Comment:
```suggestion
    test "automatically enqueues for provisioning only on create" do
```

Yes I read the tests in code review 😝

Reply:
Just checking 😀